src/langs/fr/auth.json

```diff
@@ -11,5 +11,8 @@
   "SIGN_UP_HEADER": "S'enregistrer",
   "EMAIL_FIELD_TEXT": "Email",
   "SWITCH_ACCOUNT_TEXT": "Utiliser un autre compte",
-  "NOT_SIGNED_IN_TOOLTIP": "Vous n'êtes pas connecté."
+  "NOT_SIGNED_IN_TOOLTIP": "Vous n'êtes pas connecté.",
+  "SIGN_IN_SUCCESS_TITLE": "Votre demande de connnexion a réussi",
+  "SIGN_IN_SUCCESS_TEXT": "Vous pouvez ouvrir votre boîte mail <bold>{{email}}</bold> pour utiliser votre lien de connexion personnel. L'email peut peut prendre quelques minutes à arriver.",
```

Comment:
Same comment related to `bold`.

Reply:
It's actually a custom tag defined by the translation. In auth, it is used using `components={{ bold: <strong /> }}`